app/SearchApi/BcGov.Fams3.Redis/Model/SearchRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BcGov.Fams3.SearchApi.Contracts.Person;
+using BcGov.Fams3.SearchApi.Contracts.PersonSearch;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BcGov.Fams3.Redis.Model
+{
+    public class SearchRequest
+    {
+        public Guid SearchRequestId { get; set; }
+        public Person Person { get; set; }
+        public List<ProviderProfile> Providers { get; set; }
+    }
+}
```

Comment:
shall we move this to the contract, as Dynadapter send a search request to searchApi, it also needs this class.

Reply:
Hmnnn.. this model is a composition of existing interface in the contract. Dynadapter Search API already sends request. Are we going to change  that?